src/org/jitsi/service/neomedia/MediaStreamStats.java

```diff
@@ -426,5 +420,45 @@
      * endpoint, in bits per second.
      */
     @Deprecated
-    public long getSendingBitrate();
+    long getSendingBitrate();
+
+    /**
+     * Gets the number of packets for which retransmission was requested,
+     * but they were missing from the cache.
+     * @return the number of packets for which retransmission was requested,
+     * but they were missing from the cache.
+     */
+    long getPacketsMissingFromCache();
+
+    /**
+     * Gets the number of bytes retransmitted.
+     *
+     * @return the number of bytes retransmitted.
+     */
+    long getBytesRetransmitted();
+
+    /**
+     * Gets the number of bytes for packets which were requested and found
+     * in the cache, but were intentionally not retransmitted.
+     *
+     * @return the number of bytes for packets which were requested and
+     * found in the cache, but were intentionally not retransmitted.
+     */
+    long getBytesNotRetransmitted();
+
+    /**
+     * Gets the number of packets retransmitted.
+     *
+     * @return the number of packets retransmitted.
+     */
+    long getPacketsRetransmitted();
+
+    /**
+     * Gets the number of packets which were requested and found in the
+     * cache, but were intentionally not retransmitted.
+     *
+     * @return the number of packets which were requested and found in the
+     * cache, but were intentionally not retransmitted.
+     */
+    long getPacketsNotRetransmitted();
```

Comment:
I would really like to avoid adding more code to MediaStreamStats. We already have a structure for recording statistics in TrackStats, and I think these should be added there, and the the code that currently keeps track of them should just emit events to be recorded by MediaStreamStatistics2